### DISCUSSION TITLE
Anyway to generate many images using randomized poses?

### DISCUSSION THREAD

**Discussion by jtran-developer** [Mikubill/sd-webui-controlnet#701] [Ideas]
At the moment, I'm loading one pose in, generating an image, then load a second pose in, generate, and so forth.  Would it be possible to just have it randomly load a pose from a folder by itself?  Goal is so you can generate like 100 images and have it randomly select a different pose each time.

**Comment by Jonseed** [comment]
I'm wondering the same. If it could randomly select a pose (input image) from a folder of images for each generation, that would be great. Someone made a [separate utility](https://www.reddit.com/r/StableDiffusion/comments/12cijkq/hi_all_i_made_a_small_tool_to_load_random_poses/?utm_source=share&utm_medium=web2x&context=3) to do this, but it is not built in directly to auto1111.

**Comment by myndxero** [comment]
You could look into the power of Unprompted, but requires some knowledge of basic coding.

https://github.com/ThereforeGames/unprompted

**Reply by Jonseed** [reply]
I currently use [Dynamic Prompts](https://github.com/adieyal/sd-dynamic-prompts), which is similar, to get random poses. But ControlNet would be better, achieving poses that can't be prompted with text alone.

**Comment by myndxero** [comment]
I took use dynamic, I've a really convoluted network of wildcards. It's...
Convoluted haha. But some things just aren't controllable yet.

**Comment by myndxero** [comment]
I know xyz grid has some controlnet stuff it supports, I just haven't
hadn't thought to look there until just now
